pkg/kubelet/pod_workers.go

```diff
@@ -41,6 +42,14 @@ type podWorkers struct {
 	// Currently all update request for a given pod coming when another
 	// update of this pod is being processed are ignored.
 	isWorking map[types.UID]bool
+	// Tracks the desired state of a pod if another update of a given pod is
```

Comment:
Maybe a little too verbose :) How about just mentioning that it is the last undelivered work item. A work item is undelivered if it comes in while the worker is working.

WDYT of renaming to lastUndeliveredWorkUpdate or lastWorkUpdate? Not sure if this is a cache per say.